src/components/index.ts

```diff
@@ -10,6 +10,21 @@ export * from './button';
 export { default as ButtonGroup } from './button-group';
 export * from './button-group';
 
+export { default as Fieldset } from './fieldset';
+export * from './fieldset';
+
+export { default as Link } from './link/Link';
+export * from './link/Link';
+
+export { default as LinkButton } from './link/LinkButton';
+export * from './link/LinkButton';
```

Comment:
These should not refer to the component, but to the index.ts

Reply:
See loading-indicator folder for example. The index file in that folder should export the default + everything